main.py

```diff
@@ -196,6 +218,12 @@ def scrape(self, site, depth=0):
             if not self.data.get("dry_run"):
                 resp = self.client.post("documents/", json=doc_group)
                 doc_ids.extend(d["id"] for d in resp.json())
+
+        # Upload all of the uploadable Google Drive content
+        self.client.documents.upload_directory(
+            "./out", extensions=None, access=self.access_level, projects=[self.project]
```

Comment:
We support checking by extension for regular files, google drive files should respect the file type option.